app/serializers/participant_serializer.rb

```diff
@@ -24,28 +24,35 @@ def participant_active?(user)
   active_participant_attribute :full_name, &:full_name
 
   active_participant_attribute :mentor_id do |user|
-    user.early_career_teacher_profile&.mentor&.id
+    user.teacher_profile.early_career_teacher_profile&.mentor&.id
   end
 
   active_participant_attribute :school_urn do |user|
-    user.early_career_teacher_profile&.school&.urn ||
-      user.mentor_profile&.school&.urn
+    user.teacher_profile.ecf_profile&.school&.urn
   end
 
-  attribute :participant_type do |user|
-    if user.early_career_teacher?
+  active_participant_attribute :participant_type do |user|
+    case user.teacher_profile.ecf_profile.type
+    when ParticipantProfile::ECT.name
       :ect
-    elsif user.mentor?
+    when ParticipantProfile::Mentor.name
       :mentor
     end
   end
 
   active_participant_attribute :cohort do |user|
-    user.early_career_teacher_profile&.cohort&.start_year ||
-      user.mentor_profile.cohort&.start_year
+    user.teacher_profile.ecf_profile.cohort.start_year
   end
 
   attribute :status do |user|
-    user.early_career_teacher_profile&.status || user.mentor_profile&.status || "withdrawn"
+    user.teacher_profile.ecf_profile&.status || "withdrawn"
+  end
+
+  active_participant_attribute :teacher_reference_number do |user|
+    user.teacher_profile.trn
+  end
+
+  active_participant_attribute :teacher_reference_number_validated do |user|
+    user.teacher_profile.trn.present?
```

Comment:
That field looks... excessive. Are we planning to surface some more complex things here? Cause if not, we can just rename `teacher_reference_number` to `validated_ teacher_reference_number` and convey the same meaning.

Reply:
We might want to surface TRNs that are not yet validated @kicferk1. This mirrors NPQ applications, so we'd want to change it in both places.